eco_build_images/wordpress_phpunit_test_runner.Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM php:7.4-cli
+
+# https://github.com/WordPress/phpunit-test-runner/blob/master/README.md#0-requirements
+
+# hadolint ignore=DL3008
```

Comment:
Not sure why this is needed, see: https://github.com/MariaDB/buildbot/blob/main/.hadolint.yaml#L5

Reply:
because the ignore file wasn't passed though to the hadolint container.

Reply:
ok you probably want to copy it in the workdir before the check then? Not sure what the best option is but we probably do not want to duplicate ignore lists?